### PR TITLE
Disability

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
Related to [the frs PR](https://github.com/nikhilwoodruff/frs/pull/7). This adds IIDB to the variables, and the Equality Act definition indicators, and a new ```is_standard_disabled``` variable. The ```is_standard_disabled``` variable is true for 18% of the population.

### CODE REVIEW

File: openfisca_uk/variables/person/disability.py [hunk 1]
```diff
@@ -60,3 +60,38 @@ def formula(person, period, parameters):
...
+    def formula(person, period, parameters):
+        QUALIFYING = [
+            "is_disabled",
+            "is_severely_disabled",
+            "is_enhanced_disabled",
+            "IIDB",
+            "registered_disabled",
+            "dis_equality_act_core"
+        ]
+        return add(person, period, QUALIFYING, options=[MATCH]) + person.benunit("ESA_income", period, options=[MATCH]) > 0
```
Comment (MaxGhenis) [inline openfisca_uk/variables/person/disability.py:97]: Does this mean that all persons in a benefit unit with `ESA_income>0` is classified as disabled?

Also seems like this line might be too long, and lacks a newline.

**Review by MaxGhenis — COMMENTED** [review]
Should this be linked to #47?

File: openfisca_uk/variables/person/disability.py (hunk 1 above)
Comment (nikhilwoodruff) [reply openfisca_uk/variables/person/disability.py:97]: Good catch, I'll take it out

Comment (nikhilwoodruff) [reply openfisca_uk/variables/person/disability.py:97]: Good catch, I'll take it out

**Comment by nikhilwoodruff** [comment]
Just added the black formatter check GitHub action again

**Comment by nikhilwoodruff** [comment]
Also, removed the ```debug.log``` from the repo as raised in #50 , interestingly it didn't appear in my VSCode install, seems like it was generated and protected by a [bug](https://github.com/microsoft/vscode/issues/105191).

**Pull request merged**